proxy.h

```diff
@@ -1489,60 +1489,85 @@ using merge_facade_conv_t = typename add_upward_conversion_conv<
         F::constraints.copyability != constraint_level::trivial) ?
         F::constraints.relocatability : constraint_level::none>::type;
 
+template <class LR, class CLR, class RR, class CRR>
+struct observer_ptr {
+ public:
+  explicit observer_ptr(std::remove_reference_t<LR>* ptr) : ptr_(ptr) {}
```

Comment:
`noexcept`?

Reply:
This is only intended for internal use. `requires` and `noexcept` are intended to present on the API boundary.